loader/loader.c

```diff
@@ -6860,7 +6860,6 @@ VkResult setup_loader_term_phys_dev_groups(struct loader_instance *inst) {
             }
         }
         total_count += cur_icd_group_count;
-        
```

Comment:
Not a problem to remove whitespace, just need to make sure it doesn't cause conflicts with other in flight PR's